recipes/bioconductor-purecn/meta.yaml

```diff
@@ -20,19 +20,19 @@ build:
 # Suggests: PSCBS, copynumber, BiocStyle, BiocParallel, knitr, optparse, TxDb.Hsapiens.UCSC.hg19.knownGene, org.Hs.eg.db, covr, testthat
 requirements:
   host:
-    - 'bioconductor-biocgenerics >=0.30.0,<0.31.0'
-    - 'bioconductor-biostrings >=2.52.0,<2.53.0'
+    - bioconductor-biocgenerics
+    - bioconductor-biostrings
```

Comment:
The pinnings were required by Bioconductor for proper compatibility, they can't be removed. I think BiocGenerics requires R 3.6, which will be available in conda in July.